.prettierignore

```diff
@@ -0,0 +1,10 @@
+# Prettier will format everything, except what is explicitly specified here.
+
+# npm
+node_modules/
+
+# Generated files
+dist/
+
+# Configuration
+.env
```

Comment:
`.env` wouldn't be formatted by prettier - they have explicitly [decided not to do it](https://github.com/prettier/prettier/issues/6103. So there is no need to list it in `.prettierignore` as well

Reply:
Not sure that relying on an explicit rule for this one is as obvious as it is for `node_modules`.
We have it in our `.gitignore`, so it seems easier if `.gitingore` and `.prettierignore` are identical.

Reply:
Replaced with a comment saying that `.env` is excluded by an implicit rule.